packages/@lwc/shared/src/language.ts

```diff
@@ -171,13 +172,14 @@ export function toString(obj: any): string {
         }
         return obj.toString();
     } else if (typeof obj === 'object') {
+        // Oops! This catches null and returns "[object Null]"
```

Comment:
You might want to mention that we're keeping this behavior for now due to fears of backwards compat issues. Otherwise this `Oops` could be confusing.